File: SofiForce.Models/Models/SearchModels/ItemStoreTotalSearchModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class ItemStoreTotalSearchModel : BaseSearchModel
    {
        public Int32? VendorId { get; set; }
        public Int32? ItemId { get; set; }
        public Int32? BranchId { get; set; }

        public Int32? StoreId { get; set; }



    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/SearchModels/ItemStoreTotalSearchModel.ts
import { BaseSearchModel } from "./baseSearchModel";

export interface ItemStoreTotalSearchModel extends BaseSearchModel {
    vendorId: number | null;
    vendorCode: string | null;

    itemId: number | null;
    itemCode: string | null;

    branchId: number | null;
    branchCode: string | null;

    storeId: number | null;
    storeCode: string | null;

}